src/SimpleDB/CheepDatabase.cs

```diff
@@ -4,30 +4,39 @@
 
 namespace SimpleDB;
 
-public sealed class CSVDatabase<T> : IDatabaseRepository<T>
+public sealed class CheepDatabase : IDatabaseRepository<Cheep>
 {
-    private string DatabasePath;
+    private static CheepDatabase? instance;
+    private static readonly string DatabasePath = "../../data/chirp_cli_db.csv";
     private static readonly CultureInfo CultureInfo = new("en-DE");
 
-    public CSVDatabase(string databasePath)
+    public static CheepDatabase Instance
     {
-        DatabasePath = databasePath;
+        get
+        {
+            if (instance == null)
+            {
+                instance = new CheepDatabase();
+            }
+
+            return instance;
+        }
     }
 
     /// <summary>
     /// Reads records from the CSV database, with an optional limit on the number of records returned.
     /// </summary>
     /// <param name="limit">Optional parameter to specify the maximum number of records to return.</param>
     /// <returns>An enumerable collection of records of type T.</returns>
-    public IEnumerable<T> Read(int? limit = null)
+    public IEnumerable<Cheep> Read(int? limit = null)
     {
         using StreamReader reader = new(DatabasePath);
         using CsvReader csv = new(reader, CultureInfo);
```

Comment:
Please rename the csv variable, to something more descriptive. Since this is a reader